.github/workflows/wpt_epoch.yml

```diff
@@ -35,10 +35,9 @@ jobs:
           deno-version: ${{ matrix.deno-version }}
 
       - name: Install Python
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v4
         with:
           python-version: "3.8"
-          architecture: x64
```

Comment:
architecture is x64 by default
https://github.com/actions/setup-python#supported-architectures